src/platforms/common/configuration/options.mdx

```diff
@@ -411,6 +411,30 @@ If the callback is not set, or it returns `undefined`, the default naming scheme
 
 </PlatformSection>
 
+<ConfigKey name="send-client-reports" supported={["java", "javascript", "dotnet", "python", "dart", "apple"]}>
```

Comment:
```suggestion
<ConfigKey name="send-client-reports" supported={["java", "javascript", "dotnet", "python", "dart", "apple"]} notSupported={["react-native"]}>
```

`react-native` inherits its config from `javascript` but we don''t have support for RN yet.